### DISCUSSION TITLE
《世界树的迷宫III 星海的访客》汉化笔记（一）：初探字库

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#29] [General]
# 《世界树的迷宫III 星海的访客》汉化笔记（一）：初探字库

《世界树的迷宫III 星海的访客》汉化笔记（一）：初探字库 - 来自 Xzonn 的小站

https://xzonn.top/posts/Etrian-Odyssey-III-Chinese-Localization.html

**Comment by XiaoFuDieZZZ** [comment]
大佬求一下  汉化美版心金的教程😭😭😭😭😭
